modules/coralogix-aws-shipper/main.tf

```diff
@@ -0,0 +1,260 @@
+locals {
+
+  function_name = "Coralogix-${var.integration_type}-${random_string.this.result}"
+
+  coralogix_regions = {
+    Europe    = "coralogix.com"
+    Europe2   = "eu2.coralogix.com"
+    India     = "coralogix.in"
+    Singapore = "coralogixsg.com"
+    US        = "coralogix.us"
+    US2       = "cx498.coralogix.com"
+  }
+
+  coralogix_url_seffix = "/api/v1/logs"
+
+  tags = {
+    Provider = "Coralogix"
+    License  = "Apache-2.0"
+  }
+
+  s3_suffix_map = {
+    CloudTrail    = ".json.gz"
+    VpcFlow = ".log.gz"
```

Comment:
it should be also .json.gz

Reply:
please check it from old lambda it looks like it is .json.gz might be mistake in the old one as well